CapybaraLearning/spec/dashboard_links_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../all_requires.rb'
+
+describe 'When testing demo.opensourcecms website' do
+  let(:login_page) { LoginPage.new }
+  let(:home_page) { HomePage.new }
+
+  before(:all) do
+    Capybara.configure do |config|
+      config.run_server = false
+      config.default_driver = :poltergeist
+    end
+  end
+
+  it 'Verify logging' do
```

Comment:
You do not need to include 'Verify logging' test in each spec file. Just check it once and add login to before in other specs